common/src/test/java/org/opensearch/sql/common/authinterceptors/AwsSigningInterceptorTest.java

```diff
@@ -7,8 +7,6 @@
 
 package org.opensearch.sql.common.authinterceptors;
 
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
 
```

Comment:
Probably, remove one blank line too to have only one blank between package declaration and imports.